src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Este.js pattern is: Every top element returned from component should have own css globally unique independent className, because we can't not ensure order and it would be anti-pattern anyway. For example, .todo-item, or .editable-component. 
Component children should use it as parent reference. Do you agree? Maybe I should describe it somewhere :-) Can you please update PR? I really want to merge it. Thank you!